src/elf/dynamic.rs

```diff
@@ -515,9 +515,9 @@ macro_rules! elf_dyn_std_impl {
             /// Gets the needed libraries from the `_DYNAMIC` array, with the str slices lifetime tied to the dynamic array/strtab's lifetime(s)
             pub unsafe fn get_needed<'a>(dyns: &[Dyn], strtab: *const Strtab<'a>, count: usize) -> Vec<&'a str> {
                 let mut needed = Vec::with_capacity(count);
-                for r#dyn in dyns {
-                    if r#dyn.d_tag as u64 == DT_NEEDED {
-                        let lib = &(*strtab)[r#dyn.d_val as usize];
+                for dynamic in dyns {
```

Comment:
probably `d` was a better variable choice here given `d_tag`, and `for d in dyns` seems better, but whatever.